src/test/data/JsonSerializableMcGymmyTest/duplicateFoodMcGymmy.json

```diff
@@ -0,0 +1,19 @@
+{
+  "food": [
+    {
+      "name": "Alice Pauline",
```

Comment:
I guess this is valid but should we leave this or change it to `mcspicy` or something

Reply:
HAHA can i have a serving of alice pauline?